save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestCreationComponent.kt

```diff
@@ -170,11 +170,13 @@ private fun contestCreationComponent() = FC<ContestCreationComponentProps> { pro
         className = ClassName("card")
         contestCreationCard {
             showContestTestSuitesSelectorModal(
+                props.organizationName,
```

Comment:
`contestDto.organizationName` can be used here

Reply:
Will do it

Reply:
Done